ios/sandwich/QonversionEventListener.swift

```diff
@@ -8,9 +8,9 @@
 
 import Foundation
 
-public protocol QonversionEventListener {
+@objc public protocol QonversionEventListener {
 
-  func qonversionDidReceiveUpdatedPermissions(_ permissions: BridgeData)
+  @objc func qonversionDidReceiveUpdatedPermissions(_ permissions: [String: Any])
```

Comment:
Why not `BridgeData` here?

Reply:
Because `BridgeData` is not Objective-C compatible